src/totalizer.cpp

```diff
@@ -219,6 +220,8 @@ void TTotalizer::build(MaxSATFormula *formula, vec<Lit> &lits, int64_t rhs) {
   assert(rhs >= 1 && rhs <= lits.size());
 
   if (incremental_strategy == _INCREMENTAL_NONE_ && rhs == lits.size()) {
+    DEBUG() << "Totalizer (Not incremental): Rhs is equal to lits.size()";
+    hasEncoding = true;
```

Comment:
@GoodDeeds please check if this change is appropriate.

The issue was that when RHS is equal to lits.size(), `hasEncoding` was false and update step failed(Line 179).